test/remote/gateways/remote_pay_junction_v2_test.rb

```diff
@@ -5,9 +5,10 @@ def setup
     @gateway = PayJunctionV2Gateway.new(fixtures(:pay_junction_v2))
 
     @amount = 99
-    @credit_card = credit_card('4444333322221111', month: 01, year: 2020, verification_value: 999)
+    @credit_card = credit_card('4444333322221111', month: 01, year: 2021, verification_value: 999)
```

Comment:
Nice catch on this! What do you think about bumping it up even further into the future? 🔮

Reply:
My thoughts on this are to keep it a year out so that some basic hygiene can be performed on the test data we are sending to the gateway. I'm not sure how often test cards might change, but it'll at least get whoever is working on it to go peek at the gateway's API and check for this stuff.

Reply:
@naashton The "official" test cards from PayJunction now have an expiration date of `01/2022`, and I think it's worth matching the values provided in their documentation. https://developer.payjunction.com/hc/en-us/articles/216477397-Development-Sandbox